tensorrt_llm/_torch/disaggregation/resource/cache_reuse.py

```
@@ -0,0 +1,132 @@
+from tensorrt_llm._torch.pyexecutor.resource_manager import KVCacheManager, KVCacheManagerV2
+
+from .page import AttentionLayerGroup
+from .utils import get_global_layer_ids
+
+
+class CacheReuseAdapter(ABC):
+    """Uniform prefix-reuse API over KVCacheManager V1/V2."""
+
+    @property
+    @abstractmethod
+    def enable_block_reuse(self) -> bool: ...
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)